dcos/http.py

```diff
@@ -190,6 +210,7 @@ def _request_match(expected_url, actual_url):
     if is_success(response.status_code):
         return response
     elif response.status_code == 401:
+        prompt_login = config.get_config_val("core.prompt_login", toml_config)
```

Comment:
This change seems unrelated. It should either be a second commit in this pull request with a detailed commit message describing its purpose, or a separate PR.